src/views/RecIM/components/List/Listing/Listing.module.scss

```diff
@@ -15,13 +15,22 @@
   padding: 0;
 }
 
-.redButton {
-  background-color: $secondary-red;
+.rejectButton {
+  background-color: #900000;
```

Comment:
If possible, these color codes should be in the theme file if they are colors used in multiple cases. Are these common colors?

Reply:
$secondary-red is a new more muted red that we added to the vars file. Is that adequate?

Reply:
I use $secondary-red as well, and it should already be in the vars file

Reply:
I think I was confused with what was going on, this change is up to you @silasw2

Reply:
Yea, I would say that if you feel we could benefit from having the color to be used in different places, then it should be in vars. Up to you.

I will say that it would be good to keep our color styling as consistent as possible. I'm not sure that having one specific button with a different shade of red is really the best option. It's also going to be more difficult when we try to centralize the theme colors and implement dark mode.